src/MissionCreator.sol

```diff
@@ -131,15 +128,15 @@ contract MissionCreator is Ownable, IERC721Receiver {
             splits: new JBSplit[](2) // Initialize as dynamic array
         });
         rulesetConfigurations[0].splitGroups[0].splits[0] = JBSplit({
-            percent: 100_000_000, // 10%, out of 1_000_000_000
+            percent: 179_487_180, // works out to 17.5% after 2.5% fee. 10% for liquidity, 7.5% for moondao fee (out of 1_000_000_000)
```

Comment:
I guess all 17.5% would go to the treasury multisig? And then it would be up to us to set up the pool from the treasury and set up the amm? I guess I don't have a strong conviction yet, but I would have thought this would bet set aside in the amm contract (but not used until the token is launched). How is the AMM contract set up at the moment? Can it "store funds" until used for liquidity or no? 

Also, is this the treasury on arbitrum (remember this is a different address than the mainnet one).

Reply:
yeah you're right.

i'm working on the script rn to automatically deploy the LP position. i'll hold off on merging this until that's done.